src/main/java/appeng/tile/storage/TileDrive.java

```diff
@@ -349,4 +357,78 @@ public void blinkCell(final int slot) {
     public void saveChanges(final IMEInventory cellInventory) {
         this.worldObj.markTileEntityChunkModified(this.xCoord, this.yCoord, this.zCoord, this);
     }
+
+    public void lockCells() {
+        int idx = 0;
+        for (ICellHandler cellHandler : this.handlersBySlot) {
```

Comment:
Use a normal for loop instead

Reply:
Resolved, see d64a646669a6487c86b053665044dae45a2d05be